.env.example

```diff
@@ -15,8 +15,6 @@ POLYGON_MUMBAI_API_KEY=fixme
 # devnets
 OPTIMISM_KOVAN_STAGING_PRIVATE_KEY=0xfixme
 OPTIMISM_KOVAN_STAGING_API_KEY=fixme
-OPTIMISM_KOVAN_STAGING_SQLITE_PRIVATE_KEY=0xfixme
```

Comment:
dropping mentions of "staging sqlite" since staging is now the sqlite deployment